### ISSUE TITLE
The "search_query" and "matched_keywords" data is not registered for Merino results

### ISSUE THREAD

**Issue by SoftVision-CosminMuntean** [mozilla-services/merino#244]
**[Affected Versions]:**
- Firefox Nightly 96.0a1 (Build ID: 20211124155224) 
- Firefox Beta 95.0b11 (Build ID: 20211123190150)
- Firefox Release 94.0.2

**[Affected Platforms]:**
- Windows 10 x64
- macOS 10.15.7
- Ubuntu 20.04 x64

**[Prerequisites]**
- Have the "browser.urlbar.merino.enabled" set to true.
- Have the "browser.urlbar.merino.endpointURL" set to "https://stage.merino.nonprod.cloudops.mozgcp.net/api/v1/suggest".
- Have the "browser.urlbar.quicksuggest.remoteSettings.enabled" set to "false".
- Have the "browser.urlbar.quicksuggest.dataCollection.enabled" set to "true"
- Have the "browser.ping-centre.log" set to "true".
- Have the "browser.search.region" set to "US".
- Have the browser console opened.

**[Steps to reproduce]:**
1. Open the latest Nightly/Beta build with the profile from prerequisites.
2. Open a New Tab page and type "amazon".
3. Click the Sponsored result displayed on the last position.
4. Observe the impression ping registered in the browser console.

**[Expected results]:**
- "search_query" and "matched_keywords" data is registered and contains correct values.
    
**[Actual results]:**
- "search_query" and "matched_keywords" data is not registered.

**[Notes]:**
- For the Remote Settings results the "search_query" and "matched_keywords" is correctly registered.
- Not sure if this issue is related to #242.
- Attached is a screen recording of the issue
![the search_query and matched_keywords da is not registered ](https://user-images.githubusercontent.com/19226534/143431826-bd10833c-6663-43e8-836d-8f5bdeeae59b.gif)
.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/CONSVC-1396)

**Comment by ncloudioj** [comment]
This is the expected behavior, please see more details in [Bug 1736117](https://bugzilla.mozilla.org/show_bug.cgi?id=1736117).

Tl;dr: For the impression pings of the Merino-backed suggestions in Firefox, we replace `search_query` and `matched_keywords` with `request_id`.
